Dizzy/Screens/PlaceStory/PlaceStoryVM.swift

```diff
@@ -38,26 +38,29 @@ protocol PlaceStoryVMType {
 
 final class PlaceStoryVM: PlaceStoryVMType {
     
-    let place: PlaceInfo
+    var place: PlaceInfo
     weak var delegate: PlaceStoryVMDelegate?
     weak var navigationDelegate: PlaceStoryVMNavigationDelegate?
     
     var displayedImageIndex = -1
     let delay = 1000.0
     var commentsInteractor: CommentsInteractorType
     var storiesInteractor: StoriesInteractorType
+    var placesIteractor: PlacesInteractorType
     var comments = Observable<[CommentWithWriter]>([CommentWithWriter]())
     var stories = Observable<[PlaceMedia]>([PlaceMedia]())
     let usersInteractor: UsersInteracteorType
     let user: DizzyUser
     
-    init(place: PlaceInfo, commentsInteractor: CommentsInteractorType, storiesInteractor: StoriesInteractorType, user: DizzyUser, usersInteractor: UsersInteracteorType) {
+    init(place: PlaceInfo, commentsInteractor: CommentsInteractorType, storiesInteractor: StoriesInteractorType, user: DizzyUser, usersInteractor: UsersInteracteorType, placesIteractor: PlacesInteractorType) {
         self.place = place
         self.commentsInteractor = commentsInteractor
         self.storiesInteractor = storiesInteractor
         self.user = user
         self.usersInteractor = usersInteractor
         self.storiesInteractor.getAllPlaceStories(with: place.id)
+        self.placesIteractor = placesIteractor
+        self.placesIteractor.delegate = self
```

Comment:
remove this line because of my prev explanation

Reply:
Done